node/test/unit/tessel.js

```diff
@@ -1337,10 +1353,10 @@ exports['Tessel.Pin'] = {
       test.equal(this.b.pin[pinIndex].interruptMode, 'change');
 
       // Simulate receipt of pin state changes
-      this.a.sock.read.returns(new Buffer([REPLY.ASYNC_PIN_CHANGE_N + pinIndex]));
+      this.a.sock.read.returns(new Buffer([(REPLY.ASYNC_PIN_CHANGE_N + pinIndex) | (0 << 3)]));
       this.a.sock.emit('readable');
 
-      this.b.sock.read.returns(new Buffer([REPLY.ASYNC_PIN_CHANGE_N + pinIndex]));
+      this.b.sock.read.returns(new Buffer([(REPLY.ASYNC_PIN_CHANGE_N + pinIndex) | (0 << 3)]));
```

Comment:
I guess in my mind it just seemed to make more sense to be explicit and for the tests to look consistent since they are testing the same thing (just high vs low), but I understand that it is effectively doing nothing.

Does that make sense at all?